Sources/AppStoreConnectCLI/Commands/Users/ListUsersCommand.swift

```diff
@@ -64,12 +67,18 @@ public struct ListUsersCommand: CommonParsableCommand {
             filters.append(ListUsers.Filter.visibleApps(filterVisibleApps.compactMap { $0 }))
         }
 
+        let limits = [
+          limitUsers.map(ListUsers.Limit.users),
+          limitVisibleApps.map(ListUsers.Limit.visibleApps)
+          ]
+          .compactMap { $0 }
```

Comment:
Let's align the trailing `]` with `let` and move the compact map to endpoint constructor